haskell-src/Concordium/Types/Updates.hs

```diff
@@ -840,6 +845,8 @@ data UpdatePayload
       FinalizationCommitteeParametersUpdatePayload !FinalizationCommitteeParameters
     | -- | Update the validator score parameters (chain parameters version 3)
       ValidatorScoreParametersUpdatePayload !ValidatorScoreParameters
+    | -- | Issue a new Protocol Level Token (PLT) (Concordium Protocol 9)
```

Comment:
Do we already have a new chain parameter version for P9? I'd probably reference this in the comment.

Reply:
We don't at the moment and it is not clear to me whether we will have it.

Reply:
I think we probably will have one, but this is not an update to a chain parameter as such, so it would be correct to say "from protocol version 9".